src/scripts/main.js

```diff
@@ -1 +1,96 @@
 'use strict';
+
+const createElementDiv = document.createElement('div');
+
+createElementDiv.setAttribute('data-qa', 'notification');
+
+const firstPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('click', function (event) {
+    if (event.button === 0 || event.button === 2) {
+      resolve('Second promise was resolved never rejected');
+    }
+  });
+
+  document.addEventListener('contextmenu', function (e) {
+    e.preventDefault();
+    resolve('Second promise was resolved never rejected');
```

Comment:
The promise resolves on both click and contextmenu events, which might not be the intended behavior. Ensure this is the desired logic.